src/components/molecules/TeamCard/index.jsx

```diff
@@ -0,0 +1,18 @@
+import Heading1 from '../../shared/Typography/Heading1';
+import { MeetTheTeam, Heading } from '../../atoms/TeamCard/styles';
+import Heading4 from '../../shared/Typography/Heading4';
```

Comment:
Do not make multiple individual imports for Typography.... `import {Heading1, Heading4} from ../../shared/Typography`

Reply:
Done